config/monitoring.yaml

```diff
@@ -2,14 +2,14 @@ settings:
   configdir: /etc/cloudscopes/monitor.d
   interval: 60
   provider: cloudwatch
-  key: AWS_KEY_ID 
-  secret: AWS_SECRET_KEY
+  #key: AWS_KEY_ID
+  #secret: AWS_SECRET_KEY
```

Comment:
Yes. In my experience, using IAM instance profiles is better than to fiddle around with credentials.
I think the default should be “rely on instance profile” and if one does not want to, she could simply set the credentials.